elastic/kibana
# [ResponseOps] Hyperlinks in Slack messages are broken when there is "_" or "*" in the URL

Resolves https://github.com/elastic/kibana/issues/165673

## Summary
`escapeSlack` function in the mustasche_renderer breakes the hyperlinks by wrapping the URL with backticks.
So the below markdown template does not work.

`<{{context.alertDetailsUrl}}|View alert details>`

This PR removes the code that adds backticks. 
With this change action variables with text wrapped in asterisks \*text\* will render as **text** or wrapped in underscores \_text\_ will render as _text_ .


### Checklist



### To verify

- Create a slack connector
- Create a rule that uses a slack connector and use the above markdown template
- Verify that hyperlink is working properly in slack
